File: 1сем/ОАиП лабы/Common7/IDE/ItemTemplatesCache/Web/CSharp/1049/WcfService/App_Code/IService.cs
using System;
using System.Collections.Generic;
$if$ ($targetframeworkversion$ >= 3.5)using System.Linq;
$endif$using System.Runtime.Serialization;
using System.ServiceModel;
using System.Text;

// ПРИМЕЧАНИЕ. Можно использовать команду "Переименовать" в меню "Рефакторинг", чтобы изменить имя интерфейса "$safeitemrootname$" в коде и файле конфигурации.
[ServiceContract]
public interface $safeitemrootname$
{
	[OperationContract]
	void DoWork();
}

File: 1сем/ОАиП лабы/Common7/IDE/ItemTemplatesCache/Web/VisualBasic/1049/WcfService/App_Code/IService.vb
Imports System.ServiceModel

' ПРИМЕЧАНИЕ. Можно использовать команду "Переименовать" в контекстном меню, чтобы изменить имя интерфейса "$safeitemrootname$" в коде и файле конфигурации.
<ServiceContract()>$if$ ($targetframeworkversion$ <= 3.5) _$endif$
Public Interface $safeitemrootname$

    <OperationContract()>$if$ ($targetframeworkversion$ <= 3.5) _$endif$
    Sub DoWork()

End Interface
